packages/@dcl/inspector/src/lib/logic/analytics.ts

```diff
@@ -0,0 +1,91 @@
+import { v4 as uuidv4 } from 'uuid'
+import { Analytics as SegmentAnalytics } from '@segment/analytics-node'
+import { getConfig } from './config'
+
+export enum Event {
+  ADD_ITEM = 'Add Item',
+  REMOVE_ITEM = 'Remove Item',
+  ADD_COMPONENT = 'Add Component',
+  REMOVE_COMPONENT = 'Remove Component'
+}
+
+export type Events = {
+  [Event.ADD_ITEM]: {
+    itemId: string
+    itemName: string
+  }
+  [Event.REMOVE_ITEM]: {
+    itemId: string
+    itemName: string
+  }
+  [Event.ADD_COMPONENT]: {
+    componentName: string
+    parentItemId: string
```

Comment:
i don't think we need the `itemId` in the `ADD_COMPONENT` and `REMOVE_COMPONENT` events, we could simplify a lot by removing them.